support-frontend/assets/pages/contributions-landing/components/StripeCardForm/StripeCardFormContainer.jsx

```diff
@@ -31,31 +26,14 @@ type PropTypes = {|
 |};
 
 const StripeCardFormContainer = (props: PropTypes) => {
-  // Create separate Stripe objects for REGULAR and ONE_OFF
-  const [stripeObjects, setStripeObjects] = React.useState<{[StripeAccount]: stripeJs.Stripe | null}>({
-    REGULAR: null,
-    ONE_OFF: null,
-  });
-
   const stripeAccount = stripeAccountForContributionType[props.contributionType];
   const stripeKey = getStripeKey(
     stripeAccount,
     props.country,
     props.isTestUser,
   );
 
-  React.useEffect(() => {
-    if (!props.stripeHasLoaded) {
```

Comment:
there's some more code cleanup to be done here
`stripeHasLoaded` and `setStripeHasLoaded` are no longer needed. These can also be removed from redux.
Also the `setupStripe` function in `stripe.js`

Reply:
Yeah good point - I'll get on that now

Reply:
Removed!